README.md

```diff
@@ -1,39 +1,36 @@
 <p align="center">
-  <a href="https://www.gatsbyjs.com/?utm_source=starter&utm_medium=readme&utm_campaign=minimal-starter">
-    <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
-  </a>
-</p>
+    <a href="https://github.com/process-analytics/process-analytics.dev/releases">
+        <img alt="GitHub release (latest by date including pre-releases)" src="https://img.shields.io/github/v/release/process-analytics/process-analytics.dev?label=changelog&include_prereleases"> 
+    </a>
+    <br>
+    <a href="CONTRIBUTING.md">
+        <img alt="PRs Welcome" src="https://img.shields.io/badge/PRs-welcome-ff69b4.svg?style=flat-square"> 
+    </a> 
+    <a href="CODE_OF_CONDUCT.md">
```

Comment:
```suggestion
    <a href="https://github.com/process-analytics/.github/blob/main/CODE_OF_CONDUCT.md">
```

Reply:
No need to add the path for https://github.com/process-analytics/.github/blob/main/. It works without 😉